script.js

```diff
@@ -1,121 +1,145 @@
-var input = document.querySelectorAll('input');
+var myLibrary = [];
 
-let newBook = document.querySelector('#add-new-book');
-let modal = document.querySelector('#modal');
+let library = document.getElementById('container');
+let addBookButton = document.getElementById('add-new-book');
+let form = document.getElementById('new-book');
```

Comment:
Variable names between JS and CSS classes should be the same to avoid mismatches. For example `const library` and `.library`